Stripe/STPPaymentCardTextField.m

```diff
@@ -250,6 +316,60 @@ - (void)setCvcPlaceholder:(NSString * __nullable)cvcPlaceholder {
     self.cvcField.placeholder = _cvcPlaceholder;
 }
 
+- (void)setPostalCodePlaceholder:(NSString *)postalCodePlaceholder {
+    _postalCodePlaceholder = postalCodePlaceholder.copy;
+    [self updatePostalFieldPlaceholder];
+}
+
+- (void)setPostalCodeEntryEnabled:(BOOL)postalCodeEntryEnabled {
+    _postalCodeEntryEnabled = postalCodeEntryEnabled;
+    if (postalCodeEntryEnabled
+        && !self.countryCode) {
+        self.countryCode = [[NSLocale autoupdatingCurrentLocale] objectForKey:NSLocaleCountryCode];
```

Comment:
I'm doing a setter instead of a callback in the getter because I don't want the country code to have any chance of changing between calls to the getter. This way if it is nil when you enable postal code entry, it will grab the current locale but cache it locally so that it can't change out from under us while the user is interacting with the view.